app/controllers/admin_controller.rb

```diff
@@ -0,0 +1,8 @@
+# Controller to handle mediations for admins
+class AdminController < ApplicationController
+
```

Comment:
One the specs pass RuboCop is going to complain about this empty space.  Just FYI.